10up/post-finder
# fix: Issue with Sanitising tax_query #61

### Description of the Change
Fixes the issue with sanitising tax_query with `relation` key in it.

Closes #61 

### How to test the Change
1. Create a custom post finder meta field for a term
2. For pf_render() pass below arguments
        [
		'post_type'   => 'post',
		'post_status' => 'publish',
		'tax_query'   => [
                         'relation' => 'AND',
			[
				'taxonomy' => 'category',
				'field'    => 'slug',
				'terms'    => 'videos',
				'operator' => 'NOT IN',
			],
			[
				'taxonomy' => 'category',
				'terms'    => 'images',
			],
		],
	];
3. On the term edit screen try to search for any post
4. On the browser network tab, observe the ajax call has failed/ thrown an error

### Changelog Entry
Fixed - issue with sanitising tax_query params


### Credits
Props @dhanendran 


### Checklist:
